Doc/library/stdtypes.rst

```diff
@@ -4370,8 +4370,8 @@ type, the :dfn:`dictionary`.  (For other containers see the built-in
 A dictionary's keys are *almost* arbitrary values.  Values that are not
```

Comment:
I don't like this "almost arbitrary values" wording, since several very common types (lists, dicts, sets) can't be used as dict keys. No need to change that in this PR though.